kivy/base.py

```diff
@@ -229,11 +230,32 @@ def post_dispatch_input(self, etype, me):
             if me in self.me_list:
                 self.me_list.remove(me)
 
+        if hasattr(me, 'name') and me.name == 'hover':
+            self.hover_events.append(me)
+
         # dispatch to listeners
         if not me.grab_exclusive_class:
             for listener in self.event_listeners:
                 listener.dispatch('on_motion', etype, me)
 
+        if hasattr(me, 'name') \
+                and me.name == 'hover' \
+                and len(self.hover_events) > 1:
+            current = self.hover_events[-1]
+            prev = self.hover_events[-2]
+            for w in set(prev.handled_widgets) - set(current.handled_widgets):
```

Comment:
I'm a little bit worried here, is it executed every time there is a hover? If it's possible not to create 2 set to compare every call, that would be perfect!

Reply:
Will be replaced with generator method which will yield members of the difference set.

Reply:
It won't help, and instead of just creating 2 sets and do a diff, it will also create a generator. My question was if it would be possible not to do that every time there is a dispatch.